src/infos.rs

```diff
@@ -10,3 +30,73 @@ static API_LEVEL: u8 = if cfg!(target_os = "nanos") {
 } else {
     0xff
 };
+
+#[used]
+#[link_section = "ledger.api_level"]
+static ELF_API_LEVEL: [u8; 4] = if cfg!(target_os = "nanos") {
+    *b"0\n\0\0"
+} else if cfg!(target_os = "nanox") {
+    *b"5\n\0\0"
+} else if cfg!(target_os = "nanosplus") {
+    *b"1\n\0\0"
+} else {
+    *b"255\n"
+};
+
+#[used]
+#[link_section = "ledger.target"]
+static ELF_TARGET: [u8; 8] = if cfg!(target_os = "nanos") {
+    *b"nanos\n\0\0"
```

Comment:
Are there solutions to avoid these trailing "\n\0"? It seems a bit weird having them everywhere as I guess htey won't be used by the tools reading elf sections?

Reply:
So I propose to keep it as it is currently (until better proposal)

Reply:
I'm wondering how these trailing \n and \0 will affect our tools reading metadata? Mostly Speculos and LedgerBlue for now.